Dockerfile.openshift

```diff
@@ -7,7 +7,7 @@ RUN GO111MODULE=on CGO_ENABLED=0 GOOS=${GOOS} GOPROXY=${GOPROXY} go build \
 	-o=aws-cloud-controller-manager \
 	cmd/aws-cloud-controller-manager/main.go
 
-FROM registry.ci.openshift.org/ocp/4.14:base
+FROM registry.ci.openshift.org/ocp/4.13:base
```

Comment:
This appears to be the wrong way

Reply:
++, probably missed the art rebase here

Reply:
@RadekManak and i were talking about this at lunch, there might be an issue with the rebase bot dropping commits occasionally.